clusterman/kubernetes/kubernetes_cluster_connector.py

```diff
@@ -242,6 +242,21 @@ def mark_node_migration_resource(self, event_name: str, status: MigrationStatus)
         except Exception as e:
             logger.error(f"Failed updating migration event status: {e}")
 
+    def create_node_migration_resource(
+        self, event: MigrationEvent, status: MigrationStatus = MigrationStatus.PENDING
+    ) -> None:
+        """Create CRD resource for node migration
+
+        :param MigrationEvent event: event to submit
+        :param MigrationStatus status: event status (pending by default)
+        """
+        assert self._migration_crd_api, "CRD client was not initialized"
+        try:
+            body = event.to_crd_body(labels={MIGRATION_CRD_STATUS_LABEL: status.value})
```

Comment:
Same as previous PR. Can we move status to spec?

Reply:
Answered in previous PR.